src/main/java/com/gizasystems/automationexercise/pages/CategoriesBar.java

```diff
@@ -0,0 +1,36 @@
+package com.gizasystems.automationexercise.pages;
+
+import com.shaft.driver.SHAFT;
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+
+public class CategoriesBar {
+    // Variables
+    private SHAFT.GUI.WebDriver driver;
+
+    // Locators
+    private final By category_div = By.xpath("//div[@class='panel-group category-products']");
+    public By getCategoryLinkLocator(String categoryType){
+        return  By.xpath("//a[@href='#"+categoryType+"']");
+    }
+    public By getSubCategoryLinkLocator(String categoryType,String subCategoryType ){
```

Comment:
We don't need to name the method "get", we just need the locator name as in agreed conventions.
We also don't need to make it dynamic as it's not changeable in this case.